packages/ui/EventCacheable/EventCacheable.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import hoc from '@enact/core/hoc';
+
+const EventCacheable = hoc((config = {defaultEvent: null, type: null}, Wrapped) => {
```

Comment:
This use of default parameters doesn't take advantage of the config merging the `hoc` provides. It doesn't make much difference in this case since there are no default values but, for consistency, I'd suggest using the following form which allows the author to pass partial configs which `hoc` will populate with defaults.

```
const defaultConfig = {...};
hoc(defaultConfig, (config, Wrapped) => { ... });
```